packages/editor/src/components/collab-sidebar/comments.js

```diff
@@ -136,56 +129,120 @@ export function Comments( {
 		 */
 		const calculateAllOffsets = () => {
 			const offsets = {};
-			let previousThreadData = null;
 
 			if ( ! isFloating ) {
 				return;
```

Comment:
```suggestion
				return offsets;
```

Let's return an empty offsets object here to keep state type consistent.